docs/scarpet/api/BlocksAndWorldAccess.md

```diff
@@ -419,6 +419,10 @@ Numeric function, returning the block light at position (from torches and other
 
 Numeric function, returning the sky light at position (from sky access).
 
+### `effective_light (pos)`
+
+Numeric function, returning the "real" light at position, which is affected by time and weather.
```

Comment:
What is this light level used for (by the game)?

I think that should be specified, for example something like used for spawning or whatever it's used for.

Reply:
it is used in monster.Monster.isDarkEnoughToSpawn. i think that is why monster only spawn in night